pkg/storage/tree/tree.go

```diff
@@ -60,15 +60,15 @@ func New() *Tree {
 	}
 }
 
-func (dstTrie *Tree) Merge(srcTrieI merge.Merger) {
```

Comment:
I know the linter says these names should be consistent, but I kinda feel in this particular case it's useful to have this src / dst distinction.

What do you guys thing about this? @AdrK @kolesnikovae

Reply:
I would go with consistent approach. Reader of this file should quickly learn what `t` stands for. Also, naming as `dstTree` and `srcTree` might suggests that both those vars came as an arguments and this method can work on two arbitrary trees, while it can only merge a tree provided by argument with itself and `t` denotes it ( like `self` in Python ).

Reply:
Yeah, good points, I think I'm in favor of t now